find-updates.py

```diff
@@ -108,7 +108,7 @@ def help(return_code=0):
     for name in new:
         choice = random.choice(icons)
         message = 'New Job! %s: %s' % (choice, name)
-        data = {"text": message}
+        data = {"text": message, "unfurl_links": True}
         print(data)
         response = requests.post(webhook, headers=headers, data=json.dumps(data))
         if response.status_code not in [200, 201]:
```

Comment:
why not do a `response.raise_for_status()` instead?

Reply:
This is just my preference to explicitly check for the error codes I'll allow, and then customize the message. Someone else might prefer to do it differently.

Reply:
Also just looking at the source code - it's basically checking that same status code for ranges, and then raising an exception (that we'd have to handle) if there is an issue. https://docs.python-requests.org/en/latest/_modules/requests/models/#Response.raise_for_status So I think my preference is still to just check that code and act as needed. I always thought raise_for_status() was a bit weird, maybe it's a function intended for internals or development, but possibly not ideal for a user experience?

Reply:
Oh, I don't have strong opinions, I just noticed that it could be a one-liner instead of the current code. (We do all the time in e.g. `astroquery` to handle any type of response errors. I can't recall that it ever came up as an issue with the users)

Reply:
Oh interesting! Yeah I love Python, but I'm not on team "use as few lines as possible no matter what." I think I'm on team "maximize understand-ability and readability" so in the case here, reading that "if we don't have a successful response (200/201) then print this is more clear than "raise for status" which (reading alone) I'd have to look up to see what it actually does.